gpiozero/input_devices.py

```diff
@@ -366,13 +386,11 @@ class LineSensor(SmoothedInputDevice):
     .. _CamJam #3 EduKit: http://camjam.me/?page_id=1035
     """
     def __init__(
-            self, pin=None, queue_len=5, sample_rate=100, threshold=0.5,
-            partial=False, pin_factory=None):
+            self, pin=None, pull_up=False, queue_len=5, sample_rate=100,
```

Comment:
By changing the order of the init-params, does this break backwards-compatibility for anyone who was using positional args and not keyword args?

Reply:
I'm not bothered either way (i.e. I'd be happy with this change) - I only mentioned it because I remembered @waveform80 making a big fuss about backwards-compatibility previously :wink: